### PR TITLE
replication changes

### PR DESCRIPTION
**Pull request by itsmekumari**
replication changes

### CODE REVIEW

File: src/main/java/io/cdap/e2e/pages/locators/CdfPluginPropertiesLocators.java [hunk 1]
```diff
@@ -243,4 +243,12 @@ public static By reviewAssessment() {
   public static By configureProperties() {
     return By.xpath("//h3[contains(text(), 'Configure advanced properties')]");
   }
+
+  public static WebElement selectReplicationTable(String tableName) {
+    return SeleniumDriver.getDriver().findElement(By.xpath("//div[contains(text(),'" + tableName + "')]" +
```
Comment (anup-codex) [inline src/main/java/io/cdap/e2e/pages/locators/CdfPluginPropertiesLocators.java]: Change the code as given below, it improves the readability. 
```suggestion
    String path = "//div[contains(text(),'" + tableName + "')]" + "/preceding-sibling::div/span";
    return SeleniumDriver
    .getDriver()
    .findElement(By.xpath(path));
```

File: src/main/java/io/cdap/e2e/pages/locators/CdfPluginPropertiesLocators.java [hunk 2]
```diff
@@ -243,4 +243,14 @@ public static By reviewAssessment() {
...
+
+  public static WebElement selectReplicationTable(String tableName) {
+    String path = "//div[contains(text(),'" + tableName + "')]" + "/preceding-sibling::div/span";
+    return SeleniumDriver
+      .getDriver()
+      .findElement(By.xpath(path));
+  }
+
+  @FindBy(how = How.XPATH, using = "//div[@data-cy='log-viewer-row']//div[contains(text(),'ERROR')]")
+  public static WebElement logError;
```
Comment (poojantcs) [inline src/main/java/io/cdap/e2e/pages/locators/CdfPluginPropertiesLocators.java]: Move this to LogLocators PO class

File: src/main/java/io/cdap/e2e/pages/locators/CdfPluginPropertiesLocators.java (hunk 1 above)
Comment (itsmekumari) [reply src/main/java/io/cdap/e2e/pages/locators/CdfPluginPropertiesLocators.java]: Done.

File: src/main/java/io/cdap/e2e/pages/locators/CdfPluginPropertiesLocators.java (hunk 2 above)
Comment (itsmekumari) [reply src/main/java/io/cdap/e2e/pages/locators/CdfPluginPropertiesLocators.java]: Done.

**Pull request merged**